interface-definitions/load-balancing_reverse-proxy.xml.in

```diff
@@ -110,29 +110,55 @@
                   <leafNode name="method">
                     <properties>
                       <help>HTTP method used for health check</help>
+                      <completionHelp>
+                        <list>options get post put</list>
+                      </completionHelp>
                       <valueHelp>
-                        <format>options|get|post|put</format>
+                        <format>options|head|get|post|put</format>
                         <description>HTTP method used for health checking</description>
                       </valueHelp>
                       <constraint>
-                        <regex>options|get|post|put</regex>
+                        <regex>(options|head|get|post|put)</regex>
                       </constraint>
                     </properties>
                   </leafNode>
                   <leafNode name="uri">
                     <properties>
                       <help>URI used for HTTP health check (Example: '/' or '/health')</help>
+                      <constraint>
+                        <regex>^\/([^?#\s]*)(\?[^#\s]*)?$</regex>
+                      </constraint>
                     </properties>
                   </leafNode>
-                  <leafNode name="expect">
+                  <node name="expect">
                     <properties>
                       <help>Expected response for the health check to pass</help>
-                      <valueHelp>
-                        <format>txt</format>
-                        <description>expected response (Example: 'status 200-299' or 'string success')</description>
-                      </valueHelp>
                     </properties>
-                  </leafNode>
+                    <children>
+                      <leafNode name="status">
+                        <properties>
+                          <help>Expected status code for the health check to pass</help>
+                          <valueHelp>
+                            <format>u32:200-399</format>
+                            <description>expected response (Example: 'status 200-299' or 'string success')</description>
```

Comment:
But in format 200-399

Reply:
The first word shout be uppercase in the description.

Reply:
Fixed the description